applications/app/templates/serviceWorker.scala.js

```diff
@@ -23,11 +20,11 @@ var fetchAll = function (inputs) {
     }));
 };
 
-var cachePageAndAssetResponses = function (jsonResponse, assetResponses) {
```

Comment:
i'm going to do this in a separate PR, with a bit of further rejigging (caching regex etc)